dev/ory/device_account_v0.identity.schema.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I guess the title of the file should be device_account_or_phone_v0?

Reply:
the phone part was legacy. i removed that. the device and phone schemas are separate.